src/components/dialogs/dialog/dialogHeader/dialogHeader.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { DialogRoot } from '../dialogRoot';
+import { DialogHeader, type IDialogHeaderProps } from './dialogHeader';
+
+describe('<Dialog.Header/> component', () => {
+    const createTestComponent = (props?: Partial<IDialogHeaderProps>) => {
+        const completeProps: IDialogHeaderProps = { title: 'title', ...props };
+
+        return (
+            <DialogRoot open={true}>
+                <DialogHeader {...completeProps} />;
+            </DialogRoot>
+        );
+    };
+
+    it('renders the given title and description', () => {
+        const title = 'test title';
+        const description = 'test description';
+
+        render(createTestComponent({ title, description }));
+
+        expect(screen.getByText(title)).toBeInTheDocument();
+        expect(screen.getByText(description)).toBeInTheDocument();
+
+        const dialog = screen.getByRole('dialog');
+        expect(dialog).toHaveAccessibleName(title);
+        expect(dialog).toHaveAccessibleDescription(description);
+    });
+
```

Comment:
I would add a test like `renders a back button when the showBackButton property is set to true`